lms/djangoapps/bulk_email/models.py

```diff
@@ -32,7 +32,8 @@
 SEND_TO_MYSELF = 'myself'
 SEND_TO_STAFF = 'staff'
 SEND_TO_ALL = 'all'
-TO_OPTIONS = [SEND_TO_MYSELF, SEND_TO_STAFF, SEND_TO_ALL]
+EMAIL_TARGETS = [SEND_TO_MYSELF, SEND_TO_STAFF, SEND_TO_LEARNERS, SEND_TO_COHORT, SEND_TO_ALL]
+EMAIL_TARGET_DESCRIPTIONS = ['Myself', 'Staff and instructors', 'All students', 'Specific cohort', 'All']
```

Comment:
These descriptions are what is passed back to the email history view. These lines more properly belong on the backend PR, but I wanted to make sure to highlight the change I made to the cohort option language: `s/Send to a specific cohort/Specific cohort/`